src/HandlerStack.php

```diff
@@ -44,10 +44,10 @@ class HandlerStack
      *                                                                          handler is provided, the best handler for your
      *                                                                          system will be utilized.
      */
-    public static function create(?callable $handler = null): self
+    public static function create(?callable $handler = null, array $options = []): self
```

Comment:
Do we really need to add a second parameter here?

Reply:
Yeh, so that options can be easily passed to middlewares. I anticipate we may want other options in the future, which is why I made this an array. Without this options array, if one wants to change the body summary, they'd have to create a stack themselves and add all the middlewares, rather than just passing an option to here.

Reply:
Let’s add it here when we have more and “other options”. 
At the moment, I’m happy with letting users recreate the stack themselves.

Reply:
It would already be useful for me. ;)

Reply:
If we add it now, won't it obviously be better for forwards compatibility later?

Reply:
I'm not a huge fan of this either. `HandlerStack::create` provides you a sane default stack. If you need this (in my opinion hideous) feature, create a factory function. It's literally 5 LOC.